### ISSUE TITLE
v0.11.11 causes VS code to stall endlessly when opening a lean file

### ISSUE THREAD

**Issue by evhub** [leanprover/vscode-lean#79]
Opening any lean file after the update produces a "The window is no longer responding" error. I am using VS code 1.25.1 on Windows 10 with installed lean version 3.4.1.

**Comment by gebner** [comment]
This is sadly a known problem: https://leanprover.zulipchat.com/#narrow/stream/113488-general/subject/VScode.20extension/near/130459741

I'm having a really hard time reproducing this bug locally.  (Testing on windows 10, vscode 1.25.1, lean 3.4.1, and current mathlib.)  So far, I've only managed to make vscode lag a bit...

What lean files are you opening, mathlib?  Does the vscode window respond again after some time?  (after waiting 10 minutes or so)

**Comment by evhub** [comment]
A file as simple as
```lean
def two: nat := 2
```
reproduces the problem. After waiting for a while on that file it eventually responds, but the syntax highlighting never comes in and when I try to type at all it stops responding again.

**Comment by gebner** [comment]
If you run `lean two.lean` (the file you just showed me) from the command-line, how long does it take?

**Comment by evhub** [comment]
```powershell
> measure-command {lean .\test.lean}

Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 5
Milliseconds      : 577
Ticks             : 55770396
TotalDays         : 6.45490694444444E-05
TotalHours        : 0.00154917766666667
TotalMinutes      : 0.09295066
TotalSeconds      : 5.5770396
TotalMilliseconds : 5577.0396
```

**Comment by gebner** [comment]
Looks good.  Could you please try this build of the vscode extension?
https://1drv.ms/u/s!Au1u53SHpLowhTXMzMxSQI2Jy20o
To install, go to the extension tab, click the three dots, and select "Install from VSIX"

**Comment by evhub** [comment]
I get "This item might not exist or is no longer available" when I click on your OneDrive link.

**Comment by gebner** [comment]
In the meantime @digama0 found the bug, and the fix is already released in the latest 0.11.12 version.  Plesse check if this fixes the issue for you as well.

**Comment by evhub** [comment]
Can confirm, 0.11.12 looks to be working!
